plugins/base/src/main/kotlin/renderers/html/HtmlRenderer.kt

```diff
@@ -49,8 +49,10 @@ open class HtmlRenderer(
     ) {
         div("platform-tagged") {
             node.platforms.forEach {
-                div("platform-tag ${it.platformType.name}") {
-                    text(it.platformType.key.toUpperCase())
+                val targets = it.targets.joinToString(", ")
+                div("platform-tag") {
+                    if( targets == "common" ) classes = classes + "common"
```

Comment:
"common" can be upper-case and this won't work

Reply:
Changed, is it ok?